reframe/frontend/cli.py

```diff
@@ -786,6 +793,12 @@ def _case_failed(t):
         )
         printer.verbose(f'Final number of test cases: {len(testcases)}')
 
+        if options.ci_generate:
+            with open(options.ci_generate, 'wt') as fp:
+                ci.emit_pipeline(fp, testcases)
+
+            sys.exit(0)
```

Comment:
I would prefer that this options just generated the the yaml file in stdout. Now the output is a bit irrelevant.

Reply:
I updated the output and I like it more now.